### PR TITLE
Update Writing bug reports

### PR DESCRIPTION
**Pull request by mihacicluks**
Bug reports template updated

### CODE REVIEW

File: Basics/Writing bug reports.md [hunk 1]
```diff
@@ -11,110 +11,151 @@ Each bug report, at a minimum, consists of a title and a description.
 ## Title
 
 The title can either be:
-
 - An imperative denoting what needs to be done.
 	- E.g. "Add an Info button to the login screen"
 - Or a description of the error.
 	- E.g. "The app crashes when I take a photo of myself"
 
-## Description
```
Comment (MislavO) [inline Basics/Writing bug reports.md:20]: Do we maybe want to keep "Description" subtitle? As it seems it is a border between the title and the rest of the bug report?

File: Basics/Writing bug reports.md [hunk 2]
```diff
@@ -11,110 +11,151 @@ Each bug report, at a minimum, consists of a title and a description.
...
+The device could be any type of machine that is used during testing.
+E.g. “mobile device, computer, car display, home appliance”
 
-> Device: Nexus 5X (Android 7.0), HTC M9 (Android 6.0)   
-> Build: 1.1.0-b4422 (Development)
+OS could be any piece of software the user is operating at times
+E.g. “some car OS, tv OS, Firmware of appliances if needed, mobile OS, etc.”
 
-### Header - web apps
+App version where bug occurred
```
Comment (MislavO) [inline Basics/Writing bug reports.md:27]: Maybe add here app version (or "build) and environment where bug occured - or keep the old wording - "In the "Build" field - name the build on which you managed to reproduce the issue. Add the environment when needed."

File: Basics/Writing bug reports.md [hunk 3]
```diff
@@ -11,110 +11,151 @@ Each bug report, at a minimum, consists of a title and a description.
...
-In the "Build" field - name the build on which you managed to reproduce the issue. Add the environment when needed.
+The device could be any type of machine that is used during testing.
+E.g. “mobile device, computer, car display, home appliance”
 
-> Device: Nexus 5X (Android 7.0), HTC M9 (Android 6.0)   
-> Build: 1.1.0-b4422 (Development)
+OS could be any piece of software the user is operating at times
+E.g. “some car OS, tv OS, Firmware of appliances if needed, mobile OS, etc.”
 
-### Header - web apps
```
Comment (MislavO) [inline Basics/Writing bug reports.md:31]: Do we want to add some specifics for web bugs too, or do you think it is covered enough in your version?

**Review by MislavO — CHANGES_REQUESTED** [review]
Left a few comments to check, but otherwise great : ). 
Also please check if we need any of the changes in the files that are not the "Writing bug reports.md" - gitignore etc. (we could remove those, or not merge that part, or if you do a new commit, you can remove them then.)

File: Basics/Writing bug reports.md (hunk 1 above)
Comment (mihacicluks) [reply Basics/Writing bug reports.md:20]: Updated in new commit

File: Basics/Writing bug reports.md (hunk 3 above)
Comment (mihacicluks) [reply Basics/Writing bug reports.md:31]: Updated in new commit

**Comment by endriina** [comment]
@mihacicluks 
Please remove all the .xml files and .gitignore files. Leave only Writing bug reports.md file with the changes.

**Comment by mihacicluks** [comment]
@endriina removed now

**Pull request merged**